opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -483,9 +483,13 @@ def to_json(self, indent=4):
         if self.parent is not None:
             if isinstance(self.parent, Span):
                 ctx = self.parent.context
-                parent_id = trace_api.format_span_id(ctx.span_id)
+                parent_id = "0x{}".format(
+                    trace_api.format_span_id(ctx.span_id)
+                )
             elif isinstance(self.parent, SpanContext):
-                parent_id = trace_api.format_span_id(self.parent.span_id)
+                parent_id = "0x{}".format(
+                    trace_api.format_span_id(self.parent.span_id)
```

Comment:
I'm confused by this. Doesn't format_span_id already format as hex and wouldn't this add 0x twice to the string?

Reply:
it changes the values to hex, but doesnt prepend the string w/ `0x`

Reply:
Probably a gap in my understanding but wouldn't that be inconsistent with trace id and span id values then? Seems weird that we'd render parent_id differently from span/trace ID. Are we forced to do this because we documented it this way? Sounds like a bug in the documentation then :)

Reply:
format context prepends `0x` to the trace_id and span_id
```
    @staticmethod
    @staticmethod
    def _format_context(context):
    def _format_context(context):
        x_ctx = OrderedDict()
        x_ctx = OrderedDict()
        x_ctx["trace_id"] = "0x{}".format(
        x_ctx["trace_id"] = "0x{}".format(
            trace_api.format_trace_id(context.trace_id)
            trace_api.format_trace_id(context.trace_id)
        )
        )
        x_ctx["span_id"] = "0x{}".format(
        x_ctx["span_id"] = "0x{}".format(
            trace_api.format_span_id(context.span_id)
            trace_api.format_span_id(context.span_id)
        )
        )
        x_ctx["trace_state"] = repr(context.trace_state)
        x_ctx["trace_state"] = repr(context.trace_state)
        return x_ctx
```

Reply:
This change makes the formatting consistent in parent_id as well

Reply:
Ah. Makes sense.